src/itest/java/com/iexec/worker/tee/post/compute/IntegrationTests.java

```diff
@@ -71,23 +68,25 @@ void after() {
     }
 
     @Test
-    void shouldHandleCallback() {
-        postCompute
-                .withNetworkMode(INTERNAL_NETWORK)
-                .withEnv("RESULT_TASK_ID", "0x0000000000000000000000000000000000000000000000000000000000000001")
-                .withEnv("RESULT_STORAGE_CALLBACK", "yes")
-                .withEnv("RESULT_SIGN_WORKER_ADDRESS", "0x0000000000000000000000000000000000000002")
-                .withEnv("RESULT_SIGN_TEE_CHALLENGE_PRIVATE_KEY", "0x000000000000000000000000000000000000000000000000000000000000003")
-                .withFileSystemBind(RESOURCES_DIR + "/iexec_out", "/iexec_out");
-
-        postCompute.start();
+    void shouldHandleCallback(EnvironmentVariables environment) {
+        final String taskId = "0x0000000000000000000000000000000000000000000000000000000000000001";
+        environment.set(RESULT_TASK_ID, taskId);
```

Comment:
```suggestion
        environment.set("RESULT_TASK_ID", taskId);
```
We want to test that too